cmd/install_test.go

```diff
@@ -67,8 +69,8 @@ func Test_loadPublickeyEncrypted(t *testing.T) {
 
 	tmpfile.Close()
 	_, _, err = loadPublickey(tmpfile.Name())
-	if err.Error() != expected {
-		t.Errorf("Unexpected error, got: %q, want: %q.", err.Error(), expected)
+	if _, ok := err.(*expected); !ok {
```

Comment:
Deliberately not using Go 1.13's `errors.Is` as `go.mod` defines Go 1.12 as a build version.
@alexellis Travis should probably test with Go 1.13 as well, no?

Reply:
"Deliberately not using" I'm not sure how you mean this, but it comes across quite aggressive. What are you trying to imply? Who was the author of the code, do you want to ask them?

Reply:
I decided to use the old, pre-Go-1.13, type-asserting version of `errors.Is`, i.e. `_, ok := err.(*expected); !ok` instead of `errors.Is(err, *expected)` as `k3sup`'s `go.mod` file defines Go 1.12 as a build version, so we can't rely on Go 1.13 features (`errors.Is`).

Reply:
Hey @alexellis, can this be merged? It wasn't meant to come across aggressive or anything and I'm sorry if it did.